packages/bpk-component-image/src/BpkImage.js

```diff
@@ -27,10 +28,28 @@ import STYLES from './bpk-image.scss';
 
 const getClassName = cssModules(STYLES);
 
-const Image = props => {
+type BpkImageProps = {
+  altText: string,
+  height: number,
+  src: string,
+  width: number,
+  className: ?string,
+  inView: ?boolean,
+  loading: ?boolean,
+  onLoad: ?() => void,
```

Comment:
prefer mixed return type of callbacks - otherwise consumers who use flow won't be able to do small inline functions without getting type checker errors:
```
onLoad={(e) => doSomething(e.target)}
```